.changelogs/feature_quiz-resume-akash-6.yml

```diff
@@ -0,0 +1,4 @@
+significance: minor
+type: dev
+entry: Added filter `lifterlms_begin_quiz_button_text` for updating 'Start Quiz'
```

Comment:
you didn't add this filter hook, it's there since ages: https://github.com/gocodebox/lifterlms/blob/bde9379029f517bd17ab657b3c5fb09e1af2bffc/templates/quiz/start-button.php#L56

Reply:
I think I overlooked something. Deleted now.